code/workspaces/test/test-script/10893_STS.md

```diff
@@ -399,6 +422,61 @@ c = Client("<scheduler address>")
 c
 ```
 
+#### TC2.2.4.1.4 Dask clusters can be deleted via the UI
+
+| **JIRA ID(s):** |  [NERCDL-963](https://jira.ceh.ac.uk/browse/NERCDL-963?jql=key%20in%20(NERCDL-963)) |OVERALL RESULT: | PASS |
+| ------------ | --------- | --------- | ------|
+| **Number** | **Step** | **Expected Result** | **Result** |
+| 1 | As a project admin navigate to the Dask page '/projects/project name/Dask' | Pre-existing Dask clusters are listed here | PASS |
+| 2 | Click on the three dots next to a Dask cluster | There is an option to delete the cluster | PASS |
+| 3 | Click on delete | A modal dialog/pop-up is shown asking the user to confirm or cancel the deletion | PASS |
+| 4 | Click on cancel | The cluster is not deleted and is still listed on the Dask page | PASS |
+| 5 | Click on delete again and confirm the deletion | The cluster is deleted and is no longer listed in the UI  | PASS |
+---
+
+#### TC2.2.4.1.5 Non project admins cannot delete Dask clusters
+
+| **JIRA ID(s):** | [NERCDL-963](https://jira.ceh.ac.uk/browse/NERCDL-963?jql=key%20in%20(NERCDL-963)) |OVERALL RESULT: | PASS |
+| ------------ | --------- | --------- | ------|
+| **Number** | **Step** | **Expected Result** | **Result** |
+| 1 | Navigate to the Dask page as a user who is not a project admin '/projects/project name/Dask' | Pre-existing Dask clusters are listed here | PASS |
+| 2 | Attempt to delete a Dask cluster | There is no way to delete the Dask cluster. The three dots button is not present | PASS |
+---
+
+#### TC2.2.4.1.6 Have a Snippet button for connecting to Dask cluster
+
+| **JIRA ID(s):** | [NERCDL-989](https://jira.ceh.ac.uk/browse/NERCDL-989?jql=key%20in%20(NERCDL-989)) |OVERALL RESULT: | PASS |
+| ------------ | --------- | --------- | ------|
+| **Number** | **Step** | **Expected Result** | **Result** |
+| 1 | For an existing cluster click on the three dot icon | There are two options: delete or copy snippet | PASS |
+| 2 | Click on copy snippet | There is a confirmation message shown in the UI stating the snippet has been copied to the clipboard | PASS |
+| 3 | Create a new notebook | Notebook is created | PASS |
```

Comment:
This has to be a JupyterLab notebook (not e.g. RStudio)

Reply:
Specified JupyterLab notebook in test steps